tests/ValidateSbrpTests/ValidateSbrpTests.cs

```diff
@@ -0,0 +1,170 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Packaging;
+using NuGet.Packaging.Signing;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ValidateSbrpTests;
+
+public class ValidateSbrpTests
+{
+    private string[] Packages { get; set; }
+
+    private const string SbrpAttributeType = "System.Reflection.AssemblyMetadataAttribute";
+
+    private const string VerionPattern = @"(\.\d)+([\-\w])*";
+
+    private string RepoRoot { get; set; }
+
+    public ITestOutputHelper output { get; set; }
```

Comment:
The properties can just be defined as `static readonly` fields. The members should be grouped together of like kinds. So constants should be group together, and fields together, etc.

Reply:
Out of curiosity, why is `private static readonly` preferred to `private const`?

My understanding is that const variables are implicitly static and defined at compile time, and static readonly variables are static and are defined at runtime. If `SbrpAttributeType` and `VersionPattern` are already initialized, why is static readonly preferred to const?

Reply:
I'm referring to the properties, not the existing `const` members. The comment range ended up including the `const` members because the properties surrounded them. Yes, where possible use `const` if its value is not determined at runtime.